mandatory/2-function-creation.js

```diff
@@ -45,7 +79,12 @@ function canRegister(age) {}
   )
 */
 
-function countReverse(number) {}
+function countReverse(number) {
+  while(number !==0){
+    console.log(number);
+    number--;
+  }
+}
```

Comment:
How will it behave when you give it a negative number? Should we account for that?